python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -85,8 +86,7 @@ def _make_plc_graph(
             elif values.dtype == "int64":
                 values = values.astype("float64")
         else:
-            # values = cudf.Series(cupy.ones(len(edata_x[0]), dtype="float32"))
-            values = None
+            values = cudf.Series(cupy.ones(len(edata_x[0]), dtype="float32"))
```

Comment:
Can you add a comment here? It might be good to know why this needs to be here since it uses memory and a future maintainer may not know why an algo that doesn't use weights may need it.